common/src/warp_runner/mod.rs

```diff
@@ -315,26 +315,23 @@ async fn wait_for_multipass(
 // assumes that all anyone needs from tesseract is "keypair"
 // otherwise, Tesseract::to_file probably needs to call file.sync_all()
 async fn init_tesseract(overwrite_old_account: bool) -> Result<Tesseract, Error> {
-    tokio::task::spawn_blocking(move || {
-        log::trace!("initializing tesseract");
+    log::trace!("initializing tesseract");
 
-        if overwrite_old_account {
-            // delete old account data
-            if let Err(e) = std::fs::remove_dir_all(&STATIC_ARGS.uplink_path) {
-                log::warn!("failed to delete uplink directory: {}", e);
-            }
+    if overwrite_old_account {
+        // delete old account data
+        if let Err(e) = tokio::fs::remove_dir_all(&STATIC_ARGS.uplink_path).await {
+            log::warn!("failed to delete uplink directory: {}", e);
+        }
 
-            // create directories
-            if let Err(e) = std::fs::create_dir_all(&STATIC_ARGS.warp_path) {
-                log::warn!("failed to create warp directory: {}", e);
-            }
+        // create directories
+        if let Err(e) = tokio::fs::create_dir_all(&STATIC_ARGS.warp_path).await {
```

Comment:
@dariusc93 btw, we remove the `uplink_path` (just above) but create `warp_path`. Is this on purpose or a typo? 🤔

Reply:
It is done on purpose iirc since the `uplink_path` would cover the `.user` directory (which includes the warp instance, state and other metadata) and we create the `warp_path` with `create_dir_all`, which would include the `uplink_path`. If we only remove the `warp_path`, any metadata and other bits of data in `uplink_path` would remain and could cause problems. I will admit the naming of the variables should be better so there is clarification, but could be done in a separate PR.